diff --git a/src/Converter/CodeConverter.php b/src/Converter/CodeConverter.php
index c8ec2c0..e536362 100644
--- a/src/Converter/CodeConverter.php
+++ b/src/Converter/CodeConverter.php
@@ -43,7 +43,7 @@ public function convert(ElementInterface $element)
         $lines = preg_split('/\r\n|\r|\n/', $code);
         if (count($lines) > 1) {
             // Multiple lines detected, adding three backticks and newlines
-            $markdown .= '```' . $language . "\n" . $code . "\n" . '```';
+            $markdown .= '```' . $language . "\n" . $code . "\n" . '```' . "\n\n";
         } else {
             // One line of code, wrapping it on one backtick.
             $markdown .= '`' . $language . $code . '`';
